### PR TITLE
Add of -d- group processing, apply alpha renaming convention

### LINKED ISSUE #4: Transfer all targets groups to alpha labeling
Also, make sure .R, folder, and list all match!

### PR DESCRIPTION
**Pull request by steeleb**
WAHOO!

Another round for review here. This one is arguably more convoluted in {targets} than a linear, literate workflow. Of note, you'll notice that at the group level (-c-), I've completely ignored the DSWE3 data. That is on purpose. For the time being (until I have time to catch the punt I'm making), the remainder of this workflow will focus on DSWE1 only. I don't think I noted that for the last round, so mostly an FYI.

Time might be best spent making sure that things move logically in here, and to make sure that there will be room to expand to DSWE3 in the future. i.e., did I hardcode such that it only includes DSWE1 AND the name of the function isn't specific to DSWE1? I think all changes to scripts in groups -a- through -c- are merely naming conventions. 

Obviously just tap me if you have any questions or if things need clarification. 

This also closes #4 - I'd be interested to know whether the list names bother you at all, or if I should just overhaul and give them that same looooong name as the targets group list:
Screenshot 2023-09-07 at 2 42 59 PM
The alternative would be to replace 'targets' in each of these with the full group name. 

Thank you, KW!

### CODE REVIEW

File: d_apply_handoff_coefficients/src/apply_handoffs_to7.R [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#' 
+#' 
+apply_handoffs_to7 <- function(coefficients, data_filepath) {
+  #make sure directory exists
+  dir.create("d_apply_handoff_coefficients/mid/")
+  #get some info for saving the file
+  filename <- str_split(data_filepath, "/")[[1]][4]
+  file_prefix <- str_split(filename, "_")[[1]][1]
+  file_suffix <- str_split(filename, "_v")[[1]][2]
+  DSWE <- if_else(grepl("DSWE1", data_filepath), "DSWE1", "DSWE3")
```
Comment (kathryn-willi) [inline d_apply_handoff_coefficients/src/apply_handoffs_to7.R:24]: Once we move towards collecting DSWE3, this step for selecting will need to change, right?

Comment (kathryn-willi) [reply d_apply_handoff_coefficients/src/apply_handoffs_to7.R:24]: ah, nevermind. so long as we keep the two DSWE's split into their own data_filepath's this is an unnecessary question!

**Review by kathryn-willi — APPROVED** [review]
Not much to say about this one, either! The workflow's order makes sense and I didn't find any instances of hard coded DSWE1 outside of its specific collation step!

Also, the naming conventions are great (ie, I'm fine with the a_targets_list, etc.).

File: d_apply_handoff_coefficients/src/apply_handoffs_to7.R (hunk 1 above)
Comment (steeleb) [reply d_apply_handoff_coefficients/src/apply_handoffs_to7.R:24]: preciesely! :)

**Comment by steeleb** [comment]
Thank you @kathryn-willi!

Merging, and then I have.... one more for you. This last group is what splits/assigns NW/CLP data.

**Pull request merged**